### PR TITLE
add finetune from detectron guide

### PR DESCRIPTION
**Pull request by wangg12**
#15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +192,12 @@ While the aforementioned example should work for training, we leverage the
 cocoApi for computing the accuracies during testing. Thus, test datasets
 should currently follow the cocoApi for now.
 
+## Finetuning from Detectron weights on custom datasets
+Create a script `tools/trim_detectron_model.py` like [here](https://gist.github.com/wangg12/aea194aa6ab6a4de088f14ee193fd968).
+You can decide which keys to be removed and which keys to be kept by modifying the script.
+
+Then you can simply point the converted model path in the config file by changing `MODEL.WEIGHT`.
```
Comment (fmassa) [inline README.md]: Can you also point to the relevant issue?
Something like

> For further information, please refer to https://github.com/facebookresearch/maskrcnn-benchmark/issues/15

Comment (wangg12) [reply README.md]: Sure.

File: README.md [hunk 2]
```diff
@@ -192,6 +192,14 @@ While the aforementioned example should work for training, we leverage the
...
+
+For further information, please refer to #15.
```
Comment (fmassa) [inline README.md]: I think you need to point to the link, something like

```
[#15](https://github.com/facebookresearch/maskrcnn-benchmark/issues/15)
```

Comment (wangg12) [reply README.md]: OK

**Comment by fmassa** [comment]
@DIYer22 I think that the error as we have now is the safest option at the moment.
We already log a number of things during training, and a warning could easily be missed, but training would go on but results would be worse.

**Pull request merged**